lib/python/rose/config_editor/plugin/um/widget/stash_add.py

```diff
@@ -110,19 +110,19 @@ def __init__(self, stash_lookup, request_lookup,
             get_tooltip_func=self.set_tree_tip)
         self._view.set_rules_hint(True)
         self.sort_util = rose.gtk.util.TreeModelSortUtil(
-            lambda: self._view.get_model(), 2)
+            self._view.get_model, 2)
         self._view.show()
         self._view.connect("button-press-event",
                            self._handle_button_press_event)
         self._view.connect("cursor-changed",
-                           lambda v: self._update_control_widget_sensitivity())
+                           lambda v: self._update_control_sensitivity())
```

Comment:
117-118 to just``self._view.connect("cursor-changed", self._update_control_sensitivity)`` instead?